packages/mobile/src/screens/profile-screen/ProfileInfo.tsx

```diff
@@ -93,19 +96,27 @@ export const ProfileInfo = (props: ProfileInfoProps) => {
   const { name, handle, does_current_user_follow, does_follow_current_user } =
     profile
 
-  const actionButtons = isOwner ? (
-    <EditProfileButton style={styles.followButton} />
-  ) : (
-    <>
-      {does_current_user_follow ? <SubscribeButton profile={profile} /> : null}
-      <FollowButton
-        style={styles.followButton}
-        profile={profile}
-        onPress={onFollow}
-        followSource={FollowSource.PROFILE_PAGE}
-      />
-    </>
-  )
+  const isOwner =
+    params.handle === 'accountUser' ||
+    params.handle?.toLowerCase() === accountHandle?.toLowerCase() ||
```

Comment:
possible if these are undefined we would check undefined === undefined? probs ok even so

Reply:
yeah good callout, we have a check that handle needs to be present for isOwner to actually be there later on, so should be good